src/main/java/com/es/phoneshop/model/product/PriceHistory.java

```diff
@@ -0,0 +1,45 @@
+package com.es.phoneshop.model.product;
+
+import java.math.BigDecimal;
+import java.text.SimpleDateFormat;
+import java.util.Currency;
+import java.util.Date;
+
+public class PriceHistory {
+    private Date startDate;
+    private BigDecimal price;
+    private Currency currency;
+
+    public PriceHistory () { }
+
+    public PriceHistory(Date startDate, BigDecimal price, Currency currency) {
+        this.startDate = startDate;
+        this.price = price;
+        this.currency = currency;
+    }
+
+    public String getStartDate() {
```

Comment:
Refactor this to become a usual getter method (returning Date). For formatting purposes on jsp use corresponding tags (like \<fmt:formatDate\>)